.gitignore

```diff
@@ -100,6 +100,7 @@ fabric.properties
 archives/.idea/
 
 config.json
+client_secret.json
```

Comment:
Neither .json files should be used. Just an old habit and artifact when I was first coding this up. We should definitely stick to env config. I'll delete both.